<launch>
    <include file="$(find tiago_iaslab_simulation)/launch/start_simulation.launch" output="log">
        <arg name="world_name" value="ias_lab_room_full_tables" />
    </include>

    <include file="$(find tiago_iaslab_simulation)/launch/navigation.launch" output="log">
    </include>

    <include file="$(find tiago_iaslab_simulation)/launch/apriltag.launch" output="log">
    </include>

    <node name="human_node" pkg="tiago_iaslab_simulation" type="human_node" output="screen">
    </node>

    <node name="tiago_client" pkg="group_04_a2" type="tiago_client" output="screen">
    </node>

    <node name="tiago_pose" pkg="group_04_a2" type="tiago_server" output="screen">
    </node>

    <node name="tiago_camera" pkg="group_04_a2" type="tiago_camera" output="screen">
    </node>

    <node name="tiago_arm" pkg="group_04_a2" type="tiago_arm" output="screen">
    </node>

</launch>
